asistente_ladm_col/gui/toolbar.py

```diff
@@ -45,7 +45,14 @@ def __init__(self, iface):
         self.app = AppInterface()
         self.geometry = GeometryUtils()
 
-    def build_boundary(self, db):
+    def build_boundary(self, db, use_user_selection=True):
```

Comment:
You can rename the method to `build_boundaries()`, since that's what the method does.

Reply:
Adjustment made, available in commit 513f31a